roles/openshift_logging/vars/main.yaml

```diff
@@ -1,10 +1,11 @@
 ---
 openshift_master_config_dir: "{{ openshift.common.config_base }}/master"
+default_logging_ns: openshift-logging
```

Comment:
my vote is if we aren't going to let users customize anymore, we shouldn't have a variable to allow it (we still have the old one that we will be deprecating). There isn't really a need to have this in a variable since we only use it when we set a fact and further down in this file.